content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
+
+To check your marker matches one of these types, you can use the `in` operator:
+
+```python
+marker = r.camera.see()[0]  # Only the closest marker
+
+marker.id in WALL
+>>> True 
+```
+
+To see the full range of constants, and the exact IDs they match up to, have a look at [the source](https://github.com/sourcebots/robot-api/blob/master/robot/game_specific.py).
+
+### _Wall_ Markers
```

Comment:
What's the purpose of the emphasis on "wall" here (and similarly for the following two headings)?